.readthedocs.yaml

```
@@ -1,8 +1,8 @@
 version: 2
 build:
-  os: ubuntu-20.04
+  os: ubuntu-22.04
```

If we're going to do this might as well do it properly.
```suggestion
  os: ubuntu-24.04
```

---

why not go to ubuntu-latest directly ?